api/src/test/java/org/pmiops/workbench/api/CloudTaskUserControllerTest.java

```diff
@@ -96,16 +100,13 @@ public void testSynchronizeAccess() {
     // Ideally we would use a real implementation of UserService and mock its external deps, but
     // unfortunately UserService is too sprawling to replicate in a unit test.
 
+    // we only sync 2FA users with completed 2FA
+    verify(mockUserService).syncTwoFactorAuthStatus(userA, Agent.asSystem());
+
     // we sync DUCC for all users
     verify(mockUserService).syncDuccVersionStatus(userA, Agent.asSystem());
     verify(mockUserService).syncDuccVersionStatus(userB, Agent.asSystem());
 
-    // we only sync 2FA users with completed 2FA
-    verify(mockUserService).syncTwoFactorAuthStatus(userA, Agent.asSystem());
-
-    // normally we would expect the userService sync methods to add to this count, but userService
-    // is mocked so we only see the direct calls from synchronizeUserAccess(), one per user
-    verify(mockUserService, times(2)).updateUserAccessTiers(any(), any());
```

Comment:
we don't call this method directly from `synchronizeUserAccess()` at all anymore